docs/source/appendix/faq.rst

```diff
@@ -0,0 +1,16 @@
+.. _faq:
+
+Frequently (Asked and) Answered Questions
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+How can I learn more about Cat-VRS and get involved?
+..    see :ref:'getting_involved'
```

Comment:
Is this supposed to be a link?

Reply:
probably supposed to link [here](https://github.com/ga4gh/cat-vrs/blob/c151a3b7410897c33451890c52a9a6721ae96703/docs/source/getting_involved.rst?plain=1#L1), minus the underscore vs dash typo

Reply:
oh it also should have an extra line break between 6 and 7 if you want it to (not) render as a commented-out line, rather than just the literal text

Reply:
It ended up being a problem that the 'getting_involved' was not a unique label, so it threw a fit and decided not to compile.  Which, fair enough.

Reply:
Yea, it turns out this is one place where sphinx works very different from LaTeX, because the latter won't really care and just default to the earliest copy of the label.  Which... isn't better, just that that was what I was expecting for this problem, which is why I was confounded for so long.